src/openrct2/ride/VehicleEntry.h

```diff
@@ -87,44 +87,64 @@ enum : uint32_t
     VEHICLE_SPRITE_FLAG_USE_4_ROTATION_FRAMES = (1 << 15),
 };
 
+enum VehicleSpriteGroup : uint8_t
+{
+    SlopeFlat = 0,
+    Slopes12,
+    Slopes25,
+    Slopes42,
+    Slopes60,
+    Slopes75,
+    Slopes90,
+    SlopesLoop,
+    SlopeInverted,
+    Slopes8,
+    Slopes16,
+    Slopes50,
+    FlatBanked22,
+    FlatBanked45,
+    InlineTwists,
+    Slopes12Banked22,
+    Slopes8Banked22,
+    Slopes25Banked22,
+    Slopes25Banked45,
+    Slopes12Banked45,
+    Corkscrews,
+    RestraintAnimation,
+    CurvedLiftHill,
+    Count
+};
+
+struct SpriteGroup
+{
+    uint32_t imageId{};
+    OpenRCT2::Entity::Yaw::SpritePrecision spritePrecision{};
+    bool Enabled() const
+    {
+        return spritePrecision != OpenRCT2::Entity::Yaw::SpritePrecision::None;
+    }
+};
+
+#pragma push(pack, 1)
```

Comment:
we don't need tight packing afaik.

Reply:
Is tight packing a lot less efficient than what the compiler decides?

Reply:
Its generally best to just leave the compiler to do its thing unless you really know better.